docs/known_issues.rst

```diff
@@ -388,3 +388,37 @@ this can be fixed by updating to the latest version of the ``_astropy_init.py``
 file. The root cause of this issue is that pytest now tries to pick up the
 top-level ``test()`` function as a test, so we need to make sure that we set a
 ``test.__test__`` attribute on the function to ``False``.
+
+
+Workarounds for Unspported Versions
+===================================
+
+These are problems in *earlier* versions of Astropy which are caused by
+changes to something outside of the control of Astropy itself.  Workarounds
+are provided as a way to potentially keep code that depends on older versions
+functioning.  This are to be considered "as-is" or "best effort", though, as
+older versions of Astropy cannot be supported for arbitrarily long amounts of
+time.
+
+IERS server changes
+-------------------
+
+In late 2019, the canonical server for providing the most precise earth rotation
+parameters (the USNO IERS server) was taken offline for an extended period.
+While current versions of Astropy have been updated to use a mirror server,
+older versions (i.e. prior to 3.2.3) still have as their default the server
```

Comment:
```suggestion
older versions (i.e., prior to 3.2.3) still have as their default the server
```

Reply:
I'm :+1: on this change but it is, afaik, a style choice - do we have any English style guidelines anywhere that we have to follow?  If not, I recommend that we do _not_ create them...

Reply:
We do have a guideline, thanks to @lglattly ! https://docs.astropy.org/en/latest/development/style-guide.html#abbreviations